tests/50federation/31room-send.pl

```diff
@@ -0,0 +1,93 @@
+test "Outbound federation can send events",
+   requires => [ local_user_fixture(), $main::INBOUND_SERVER ],
+
+   do => sub {
+      my ( $user, $inbound_server ) = @_;
+
+      my $local_server_name = $inbound_server->server_name;
+      my $datastore         = $inbound_server->datastore;
+
+      my $creator = '@50fed:' . $local_server_name;
```

Comment:
It's a (in fact the) user on the "internal" fake federation server that SyTest creates. There isn't another way to create those currently.